fs/io_zenfs.cc

```diff
@@ -220,15 +220,22 @@ ZoneFile::~ZoneFile() {
     zone->used_capacity_ -= (*e)->length_;
     delete *e;
   }
-  CloseWR();
+  IOStatus status = CloseWR();
```

Comment:
We use "s" for status everywhere else, so please use it here as well (and in the other places you add io status variables)

Reply:
Just trying to introduce some best practice. Using descriptive names for variables carries intent more clearly for the reader.

Reply:
s is fine and used everywhere else